forms-text/opt-out.txt

```diff
@@ -1,3 +1,6 @@
+
+This MIT Authors' Opt-In Open Access License was signed on: [>optOutDateTimeStamp<].
```

Comment:
I think this should be "Opt-Out Open Access License" instead of "Opt-In" since this is the email for opting out.